src/components/accessRequests/accessRequestsList/accessRequestModal.tsx

```diff
@@ -45,13 +45,14 @@ const AccessRequestModal = (props: AccessRequestModalProps) => {
   const [disabledButtons, setDisabledButtons] = useState(false);
   const [selectedIVA, setSelectedIVA] = useState("");
   const { showMessage } = useMessages();
+  const accessRequest = props.accessRequest;
 
   async function handleButtonClickAccess(status: "allowed" | "denied") {
-    if (props.accessRequest === undefined || props.userId === undefined) {
+    if (accessRequest === undefined || props.userId === undefined) {
       return null;
     }
     setDisabledButtons(true);
-    const url = new URL(`access-requests/${props.accessRequest?.id}`, ARS_URL);
+    const url = new URL(`access-requests/${accessRequest?.id}`, ARS_URL);
```

Comment:
```suggestion
    const url = new URL(`access-requests/${accessRequest.id}`, ARS_URL);
```

the undefined case has been already covered above

Reply:
Done